themes-default/slim/src/components/display-show.vue

```diff
@@ -265,26 +283,23 @@ export default {
             $.tablesorter.columnSelector.attachTo($('#showTable, #animeTable'), '#popover-target');
         });
 
-        // Moved and rewritten this from displayShow. This changes the button when clicked for collapsing/expanding the
-        // Season to Show Episodes or Hide Episodes.
-        $('.collapse.toggle').on('hide.bs.collapse', function() {
-            const reg = /collapseSeason-(\d+)/g;
-            const result = reg.exec(this.id);
-            $('#showseason-' + result[1]).text('Show Episodes');
-            $('#season-' + result[1] + '-cols').addClass('shadow');
-        });
-        $('.collapse.toggle').on('show.bs.collapse', function() {
-            const reg = /collapseSeason-(\d+)/g;
-            const result = reg.exec(this.id);
-            $('#showseason-' + result[1]).text('Hide Episodes');
-            $('#season-' + result[1] + '-cols').removeClass('shadow');
-        });
-
-        // Generate IMDB stars
-        $('.imdbstars').each((index, element) => {
-            $(element).html($('<span/>').width($(element).text() * 12));
+        // Changes the button when clicked for collapsing/expanding the season to show/hide episodes
+        document.querySelectorAll('.collapse.toggle').forEach(element => {
+            element.addEventListener('hide.bs.collapse', () => {
+                // On hide
+                const reg = /collapseSeason-(\d+)/g;
+                const result = reg.exec(this.id);
+                $('#showseason-' + result[1]).text('Show Episodes');
+                $('#season-' + result[1] + '-cols').addClass('shadow');
+            });
+            element.addEventListener('show.bs.collapse', () => {
+                // On show
+                const reg = /collapseSeason-(\d+)/g;
+                const result = reg.exec(this.id);
+                $('#showseason-' + result[1]).text('Hide Episodes');
+                $('#season-' + result[1] + '-cols').removeClass('shadow');
+            });
         });
-        attachImdbTooltip(); // eslint-disable-line no-undef
 
         // Get the season exceptions and the xem season mappings.
         getSeasonSceneExceptions();
```

Comment:
Need to fix this function now,
this is the console error, on the first line of the function.

```js
        getSeasonSceneExceptions() {
            // this line VVV
            const indexerName = document.querySelector('#indexer-name').value;
            const seriesId = document.querySelector('#series-id').value;
```

```
[Vue warn]: Error in mounted hook: "TypeError: Cannot read property 'value' of null"

found in

---> <DisplayShow> at src/components/display-show.vue
       <Root>
warn @ webpack:///./node_modules/vue/dist/vue.esm.js?:616

TypeError: Cannot read property 'value' of null
    at VueComponent.getSeasonSceneExceptions (webpack:///./src/components/display-show.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options:488)
    at VueComponent.mounted (webpack:///./src/components/display-show.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options:326)
    at callHook (webpack:///./node_modules/vue/dist/vue.esm.js?:2959)
    at Object.insert (webpack:///./node_modules/vue/dist/vue.esm.js?:4146)
    at invokeInsertHook (webpack:///./node_modules/vue/dist/vue.esm.js?:5962)
    at Vue.patch [as __patch__] (webpack:///./node_modules/vue/dist/vue.esm.js?:6187)
    at Vue._update (webpack:///./node_modules/vue/dist/vue.esm.js?:2686)
    at Vue.updateComponent (webpack:///./node_modules/vue/dist/vue.esm.js?:2811)
    at Watcher.get (webpack:///./node_modules/vue/dist/vue.esm.js?:3182)
    at new Watcher (webpack:///./node_modules/vue/dist/vue.esm.js?:3169)
logError @ webpack:///./node_modules/vue/dist/vue.esm.js?:1776
```